multichain-testing/scripts/ymax-tool.ts

```diff
@@ -467,7 +472,10 @@ const main = async (
   }
 
   if (values.getCreatorFacet) {
-    await walletStore.savingResult('creatorFacet', () => yc.getCreatorFacet());
+    const { contract } = values;
+    const creatorFacetKey =
+      contract === 'ymax0' ? 'creatorFacet' : `creatorFacet-${contract}`;
```

Comment:
This suggests that somebody is trying to use the same account for `ymaxControl` for both `ymax0` and `ymax1`. Let's not try to do that.

Reply:
btw... if I were going to stuff the contract name in there, I'd like to see the boardId too.